ui/src/app/components/popups.tsx

```diff
@@ -6,7 +7,7 @@ import onClickOutside from 'react-onclickoutside';
 
 const styles = {
   tooltip: {
-    background: 'black', color: 'white',
+    background: 'black', color: colors.white,
```

Comment:
Opt: While you're changing this, should this be black? Or `colors.dark` or something similar?